src/modules/bookmark/components/BookmarksListModal/BookmarkModalWrapper.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import styles from './bookmarks.css';
+import styles from '@bookmark/components/BookmarksListModal/BookmarkListModal.css';
```

Comment:
@soroushm This is in the same folder as the CSS import and should be a relative import

Reply:
good catch